src/messageschannel.cpp

```diff
@@ -152,26 +163,32 @@ void MatrixMessagesChannel::processMessageEvent(const QMatrixClient::RoomMessage
     qDebug().noquote() << Q_FUNC_INFO << "Process message" << doc.toJson(QJsonDocument::Indented);
     Tp::MessagePart header;
     header[QStringLiteral("message-token")] = QDBusVariant(event->id());
-    header[QStringLiteral("message-sent")]  = QDBusVariant(event->timestamp().toMSecsSinceEpoch() / 1000);
```

Comment:
This looks like move for the movement itself. The only justified move is 'message-type' IMO.
The reformat violates Qt coding style but meh, let's keep it if you wish :).

Reply:
I'm actually really into follow your advice and follow Qt coding style :+1: Sorry for current state :confused: 
I'm ready to mark it as resolved currently but add to some todo and try [uncrustify](https://gist.github.com/a-andreyev/87033eb72696afedbee6bcbf83fb2def)

Reply:
removed fancy spaces at `sendMessage` part

Reply:
You still moved 'sent' and 'received' keys. I like the decision to move the 'type' key next to received though.